src/chapter2.adoc

```diff
@@ -104,23 +104,33 @@ The table format is described in <<RQSC_TABLE>>.
 |===
 |Field {set:cellbgcolor:green}|Byte Length|Byte Offset|Description
 4+<|{set:cellbgcolor:!} *Header*
-<|- Resource Type               |1  |0 <a|
+<|- Resource Type               |1  |0  <a|
                                             - 0 - Cache
                                             - 1 - Memory
                                             - 2 - 0x7F - Reserved
                                             - 0x80 - 0xFF - Vendor Specific
-<|- Length                      |1  |1 <a| Length of this specific Resource Structure.
+<|- Reserved                    |1  |1  <a|Reserved. Must be 0
+<|- Length                      |2  |2  <a| Length of this specific Resource Structure.
                                             Length includes the Resource Specific
-                                            Data bytes as well. If length is set to 16,
+                                            Data bytes as well. If length is set to 20,
                                             then, it indicates there is no resource
                                             specific data available for this structure.
-<|- Resource Flags              |2  |2 <a|Resource Type Specific flags. Refer to
+<|- Resource Flags              |2  |4  <a|Resource Type Specific flags. Refer to
```

Comment:
Why do we need these flags? We are already telling raw bandwidth is valid only for memory resource type.

Reply:
Since the raw bandwidth is an optional resource specific data, we are indicating here that the raw bandwidth is valid or not.

Reply:
"If a resource type is not identified below, then there is no Resource Specific Data defined
for that resource type and the Length of the Resource Structure must be set to 20." 
This statement under "Table 8. Resource Specific Data" indicates me that if the length is > 20 for memory resource type, Raw Bandwidth per block is valid. Why would some one  add this resource specific data field for memory resource type but indicate it as invalid?

Reply:
With the current definition, we are making the reporting of Raw Bandwidth per Block as optional. I think what you are saying is that the fact that the length field is called out as > 20 by itself is indicating that the Raw Bandwidth per block is Valid. So, why also add a flag for it? Did I get that right?

If I got that right, then I agree it kind of makes sense and for this purpose, the flag is redundant. But, if future if there's more than one element available in a resource specific data, we may want to fix the resource specific data structure size and allow the flags to be used to indicate which of the fields are valid vs. not as one usage. Of course, we don't have to contemplate those right now.

I can remove the flags usage for memory structures and keep it all reserved for future use for now.

Reply:
Yeah..Having flags field is good and can be kept for future use. But this particular flag seems unnecessary.